share/templates.d/99-generic/runtime-install.tmpl

```diff
@@ -28,6 +28,8 @@ installpkg grubby
     ## https://bugzilla.redhat.com/show_bug.cgi?id=2011615
     ## bfa-firmware contains only obsolete files - see
     ## https://bugzilla.redhat.com/show_bug.cgi?id=2152202
+    ## qcom-firmware we pull in again later but *only* on aarch64, it is
```

Comment:
By later you mean later in the script or "we'll deal with this later" because we do need this as we support QCom devices where the lack of firmware will break them so just checking.

Reply:
later in the script, in the other added line.

Reply:
agh, it looks like I didn't save the file again before sending the PR, so the new line is missing. :D will fix